### ISSUE TITLE
Issue with test_simkaMin.py

### ISSUE THREAD

**Issue by shayandoust** [GATB/simka#12]
Hello there,

I'm running into an issue when executing test_simkaMin.py after the package builds. I have tested this both with the debian package I am building as well as a fresh copy of simka untouched from this github repository. I am getting the following issue that does not result in a 100% success rate of this test:
```
...
python  ../../simkaMin/simkaMin.py -in  ../../example/simka_input.txt  -out __results__/k21_filter_0-1000_n1 -nb-cores 1 -max-memory 100  -kmer-size 21 -nb-kmers 1000 -bin ../../build/bin/simkaMinCore  -max-reads 0 -filter 
	- TEST ERROR:    mat_presenceAbsence_jaccard.csv
res
;A;B;C;D;E
A;0.000000;0.780808;0.940741;0.780808;0.446000
B;0.000000;0.000000;0.733333;0.000000;0.873737
C;0.000000;0.000000;0.000000;0.733333;0.970370
D;0.000000;0.000000;0.000000;0.000000;0.873737
E;0.000000;0.000000;0.000000;0.000000;0.000000

truth
;A;B;C;D;E
A;0.000000;0.783000;0.984000;0.783000;0.446000
B;0.000000;0.000000;0.918000;0.000000;0.875000
C;0.000000;0.000000;0.000000;0.918000;0.992000
D;0.000000;0.000000;0.000000;0.000000;0.875000
E;0.000000;0.000000;0.000000;0.000000;0.000000
```

This is currently blocking the completion of the debian package[1]. If there are any recommendations or remedies of how this can be patched or fixed upstream, I'd greatly appreciate it.

Many thanks,
Shayan Doust

[1]: https://salsa.debian.org/med-team/simka

**Comment by shayandoust** [comment]
Any news or progress regarding this?

**Comment by clemaitre** [comment]
Hi Shayan,

thank you for pointing this out.
I have corrected this test in the latest commit 85bd51f.

Best,
Claire
